Source/towners.h

```diff
@@ -33,7 +33,7 @@ void TownWitch();
 void TownBarMaid();
 void TownCow();
 void ProcessTowners();
-ItemStruct *PlrHasItem(int pnum, int item, int &i);
+ItemStruct *PlrHasItem(int pnum, int item, int *i);

```

Comment:
We could have C++-only overload to reduce the number of merge conflicts:

```c++
#ifdef __cplusplus
inline ItemStruct *PlrHasItem(int pnum, int item, int &i) { return PlrHasItem(pnum, item, &i); }
#endif
```

Reply:
@mewmew i will let this one be up to you :)

Reply:
I think that this is "not an issue until it becomes one". Essentially, the Devilution repo is starting to be more and more stable, so we won't have to many merge conflicts going forward. So, in favour of keeping the code base simple, I'd prefer to avoid using overloading (even if it is technically a solution). At least those are my thoughts, lets keep it as is. We can re-evaluate this decision if we notice that these lines cause an over abundant number of merge conflicts going forward.

Cheers,
Robin